Swat/SwatEntry.php

```diff
@@ -1,80 +1,77 @@
 <?php
 
 /**
- * A single line text entry widget
+ * A single line text entry widget.
  *
- * @package   Swat
  * @copyright 2004-2023 silverorange
  * @license   http://www.gnu.org/copyleft/lesser.html LGPL License 2.1
  */
 class SwatEntry extends SwatInputControl implements SwatState
 {
-    // {{{ public properties
-
     /**
-     * Entry value
+     * Entry value.
      *
      * Text content of the widget, or null.
      *
      * @var string
      */
-    public $value = null;
+    public $value;
 
     /**
-     * Input size
+     * Input size.
      *
      * Size in characters of the HTML text form input, or null.
      *
-     * @var integer
+     * @var int
      */
     public $size = 50;
 
     /**
-     * Maximum length
+     * Maximum length.
      *
      * Maximum number of allowable characters in HTML text form input, or null.
      *
-     * @var integer
+     * @var int
```

Comment:
Should be ?int

Reply:
Changing it to `?int` in the docblock would be fine.  We get into trouble by defining it with:
```php
public ?int $maxlength;
```
In that case, we need to give it a `= null` initial value (or `= 0` or something reasonable).  Otherwise, you get:
```
PHP Fatal error:  Uncaught Error: Typed property SwatEntry::$maxlength must not be accessed before initialization 
```

There are _many_ instances of this.  Making the changes is a bit unwieldy, and should probably be it's own PR.